config/initializers/session_store.rb

```diff
@@ -1,4 +1,4 @@
 # Be sure to restart your server when you modify this file.
 
-Rails.application.config.session_store :cookie_store, key: '_orangelight_session',
-                                                      expire_after: 30.minutes
+# Rails.application.config.session_store :cookie_store, key: '_orangelight_session',
+#                                                       expire_after: 30.minutes
```

Comment:
It's there as a reminder for https://github.com/pulibrary/orangelight/issues/585